test_runner/fixtures/benchmark_fixture.py

```diff
@@ -120,6 +121,27 @@ def record_duration(self, metric_name):
 
         self.results.record(self.request.node.name, metric_name, end - start, 's')
 
+    def get_io_writes(self, pageserver) -> int:
+        """
+        Fetch the "cumulative # of bytes written" metric from the pageserver
+        """
+        # Fetch all the exposed prometheus metrics from page server
+        all_metrics = pageserver.http_client().get_metrics()
+        # Use a regular expression to extract the one we're interested in
+        writes = re.search(r"pageserver_disk_io_bytes{io_operation=\"write\"} (\d+)", all_metrics)
```

Comment:
> \d+

Technically, all Prometheus values are floats, does `d` cover that (including separator)?

Not really important since we don't need exact string match, but worth a notice.